storage/storage_helpers.hpp

```diff
@@ -13,7 +13,7 @@ namespace storage
 class CountryInfoGetter;
 class Storage;
 
-TMwmSize constexpr kMaxMwmSizeBytes = 100 /*Mb*/ * 1024 * 1024;
+TMwmSize constexpr kMaxMwmSizeBytes = 115 /*Mb*/ * 1024 * 1024;
```

Comment:
US_California_LA.mwm                               03-May-2017 07:57           162074342
So, 115Mb is not enough, right?